README.md

```diff
@@ -1,5 +1,56 @@
 # Prosemirror JDita
-[![Build Status](https://travis-ci.com/evolvedbinary/prosemirror-jdita.svg?branch=main)](https://travis-ci.com/evolvedbinary/prosemirror-jdita)
+[![Node.js Version](https://img.shields.io/node/v-lts/prosemirror-jdita)](https://nodejs.org)
+[![Npm Package Version](https://img.shields.io/npm/v/prosemirror-jdita)](https://www.npmjs.com/package/prosemirror-jdita)
 [![Coverage Status](https://coveralls.io/repos/github/evolvedbinary/prosemirror-jdita/badge.svg?branch=main)](https://coveralls.io/github/evolvedbinary/prosemirror-jdita?branch=main)
 
 This tool generates Prosemirror documents from JDita objects. It also provides Schema Definition for proper display and editing of JDita data.
+
+# Usage
+Add the library to your project using npm or yarn
+
+```shell
+npm install prosemirror-jdita
```

Comment:
Let's just use `yarn` as that is what we previously used - I don't want to add any more complexity right now

Reply:
This is not meant for Us, these instructions are for whomever wants to use the library

Reply:
Okay, I see now. Thanks.